### DISCUSSION TITLE
Link to projects

### DISCUSSION THREAD

**Discussion by janwilmake** [community/community#14884] [Mobile]
There is no reference in the app (on iPad) to projects. It would be great to have a "projects" tab under "my work". I understand that this could be a big section to make...

I would already be very happy with a link to the projects tab when you click a repository. This link could just open a browser with the projects connected to the repo.

This would really improve usability of the GitHub app on my iPad since I'm using the "projects" functionality every day but now I have to manually switch app and navigate there instead of using the app, which is a big hassle.

**Comment by alcere** [comment]
Hi @CodeFromAnywhere! I am happy to share that we are currently working on some great updates related to projects! Please keep tuned for updates!

**Comment by candyho** [comment]
Hey👋just want to inform you we've just shipped [view and edit projects(beta) custom fields]( https://github.blog/changelog/2022-05-09-view-and-edit-projectsbeta-custom-fields-on-github-mobile/) to both iOS and Android last week! You can now edit custom text, number, date, and single-select fields for issues and pull requests that belong to the new projects experience from within the triage sheet. Try it out and let us know what you think!😄
